stock_location_tank/models/stock_move.py

```diff
@@ -0,0 +1,25 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, models
+from odoo.exceptions import UserError
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    @api.constrains("location_dest_id")
+    def _check_state(self):
```

Comment:
This method is almost identical to the one in `stock.move.line ` you should be able to remove one of them and call one from the other